hackathons.md

```diff
@@ -0,0 +1,563 @@
+---
+title: CockroachDB Hackathon Guide
+summary: Get Up & Running with CockroachDB at a Hackathon
+toc: false
+toc_not_nested: true
+twitter: true
+no_sidebar: true
+---
+
+Hello, hackers! Here you'll find everything you need to get up and running with CockroachDB. While this guide is lengthy, it includes not only details about deploying CockroachDB, but also helps you avoid common pitfalls when using a database.
+
+For the sake of simplicity, this guide assumes you're using:
+
+- Node.js
+- Hand-written SQL statements (instead of using an ORM)
+
+However, a lot of this information is easy to adapt if you're using different technology.
+
+{{site.data.alerts.callout_danger}}If you run into any issues, hit up the Cockroach Labs booth or check out our <a href="https://www.cockroachlabs.com/docs/stable/troubleshooting-overview.html">troubleshooting documentation</a>.{{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## Quick Start: Code Sample Repo
```

Comment:
I'm a little confused about how this "quick start" ties into the rest of the guide. If starting cockroach is a prerequisite, would it make sense to move this after that, perhaps as a step?